lib/simctl.js

```diff
@@ -523,38 +524,39 @@ async function getDevices (forSdk = null) {
      *   }
      * }
      */
-    devices = {};
-    for (let [sdkName, entries] of _.toPairs(JSON.parse(stdout).devices)) {
-      if (sdkName.indexOf('iOS') !== 0) {
+    for (const [sdkName, entries] of _.toPairs(JSON.parse(stdout).devices)) {
+      const versionMatch = new RegExp(`^${_.escapeRegExp(platform)}\\s+(\\S+)`, 'i').match(sdkName);
+      if (!versionMatch) {
         continue;
       }
-      let sdk = sdkName.replace('iOS ', '');
-      devices[sdk] = entries.map((el) => {
+      devices[versionMatch[1]] = entries.map((el) => {
         delete el.availability;
-        return {...el, sdk};
+        return {
+          ...el,
+          sdk: versionMatch[1]
+        };
       });
     }
   } catch (err) {
     log.debug(`Unable to get JSON device list: ${err.message}`);
-    log.debug('Falling back to manually parsing');
+    log.debug('Falling back to manual parsing');
     devices = await getDevicesByParsing();
```

Comment:
So nothing but iOS if we get here?

Reply:
You should be able to parse for other platforms as well.

Reply:
Never mind. Just saw it was updated.